data/fintype.lean

```diff
@@ -242,26 +242,26 @@ instance (α : Type u) (β : Type v) [fintype α] [fintype β] : fintype (α ⊕
   fintype.card (α ⊕ β) = fintype.card α + fintype.card β :=
 by rw [sum.fintype, fintype.of_equiv_card]; simp
 
-lemma fintype.card_le_of_injective [fintype α] [fintype β] (f : α → β) 
+lemma fintype.card_le_of_injective [fintype α] [fintype β] (f : α → β)
   (hf : function.injective f) : fintype.card α ≤ fintype.card β :=
 by haveI := classical.prop_decidable; exact
 finset.card_le_card_of_inj_on f (λ _ _, finset.mem_univ _) (λ _ _ _ _ h, hf h)
 
 lemma fintype.card_eq_one_iff [fintype α] : fintype.card α = 1 ↔ (∃ x : α, ∀ y, y = x) :=
-by rw [← fintype.card_unit, fintype.card_eq]; 
-exact ⟨λ ⟨a⟩, ⟨a.symm unit.star, λ y, a.bijective.1 (subsingleton.elim _ _)⟩, 
-λ ⟨x, hx⟩, ⟨⟨λ _, unit.star, λ _, x, λ _, (hx _).trans (hx _).symm, 
+by rw [← fintype.card_unit, fintype.card_eq];
+exact ⟨λ ⟨a⟩, ⟨a.symm unit.star, λ y, a.bijective.1 (subsingleton.elim _ _)⟩,
+λ ⟨x, hx⟩, ⟨⟨λ _, unit.star, λ _, x, λ _, (hx _).trans (hx _).symm,
```

Comment:
Syntactically, the two `λ` are inside the same `⟨...⟩` pair. You should align them to highlight that.

Reply:
Also, `exact` should be more indented than `rw`

Reply:
These aren't even part of the PR, they're just whitespace changes, due to me having the `trimTrailingWhitespace` setting in VScode turned on.

Reply:
He's still right though. I guess I missed these in the first pass. I use a special indent rule for `byrw;

Reply:
Hopefully corrected now.